docker/reqmgr2ms-unmerged/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM reqmgr2ms:latest
```

Comment:
@todor-ivanov it is bad practice to start from latest since it does not allow to reproduce a potential issue with image since latest can change any time. I suggest that you apply specific tag of the image you start with. See how it is done for reqmgr2ms image which use specific tag of cmsweb one.

Reply:
thanks @vkuznet I know how to put the tag . Can you save me some time digging and point me how to tag the `reqmgr2ms` or to take it's tag in case it is already done.

Reply:
All cmsweb images appear on docker hub, just use it to search for your favorite service and see which tags are there and what was the latest build, e.g. https://hub.docker.com/repository/docker/cmssw/reqmgr2ms

Reply:
super  ! Thanks! I just did it please  take a look at my latest commit